prow/plugins/releasenote/releasenote.go

```diff
@@ -451,3 +463,60 @@ func labelsSet(labels []github.Label) sets.String {
 	}
 	return prLabels
 }
+
+// editReleaseNote is used to edit the top level release note.
+// Since the edit itself triggers an event we don't need to worry
+// about labels because the plugin will run again and handle them.
+func editReleaseNote(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) error {
+	org := ic.Repo.Owner.Login
+	repo := ic.Repo.Name
+	user := ic.Comment.User.Login
+
+	isMember, err := gc.IsMember(org, user)
+	if err != nil {
+		return fmt.Errorf("unable to fetch if %s is an org member of %s: %v", user, org, err)
+	}
+	if !isMember {
+		return gc.CreateComment(
+			org, repo, ic.Issue.Number,
+			plugins.FormatResponseRaw(ic.Comment.Body, ic.Issue.HTMLURL, user, "You must be an org member to edit the release note."),
+		)
+	}
+
+	newNote := getReleaseNote(ic.Comment.Body)
+	if len(newNote) == 0 {
```

Comment:
Nit: When checking string for emptyness, it is IMHO nicer to compare against the empty string vs len, because then if someone reads the code its immediately obvious that what you are comparing (here: `newNote`) is of type string